tutorials/sar-basics/index.md

```diff
@@ -0,0 +1,181 @@
+  ['Germany', 'Poland', 'Czechia', 'Slovakia', 'Austria', 'Hungary']   // European
+  //['Ethiopia', 'Somalia', 'Kenya']  // Near the Equator
+  // ['India']  // 12 days revisit, mostly
+  )).union().first()).geometry().bounds();  // get bounds to simplify geometry
+
+  
+// Define time period
+var start_date = '2020-06-01';
+var end_date = '2020-06-16';
+
+// Select S1 IW images in area of interest and time period
+var s1 = ee.ImageCollection('COPERNICUS/S1_GRD').
+  filterMetadata('instrumentMode', 'equals', 'IW').
+  filterBounds(aoi).
+  filterDate(start_date, end_date);
+  
+
+var cnt = ee.FeatureCollection(s1).
+  // uncomment next line if you want to be specific
+  //filterMetadata('orbitProperties_pass', 'equals', 'ASCENDING').  
+  reduceToImage(['system:index'], ee.Reducer.count());
+  
+var upper = cnt.reduceRegion({reducer: ee.Reducer.max(), geometry: aoi, scale: 1000, bestEffort: true});
+
+print("Max count:", upper.get('count'));
+
+// Define red to green pseudo color palette
+var red2green = ['a50026','d73027','f46d43','fdae61','fee08b','ffffbf','d9ef8b','a6d96a','66bd63','1a9850','006837'];
+
+Map.addLayer(cnt.updateMask(cnt.gt(0)).clip(aoi), {min:1, max:upper.get('count').getInfo(), palette: red2green}, 'All');
+Map.centerObject(aoi,5);
+
+```
+You can change the scripts parameters to find out how S1 coverage varies across different parts of the World.
+You will notice that coverages get denser towards the poles, as orbits which are adjacent at the Equator start to overlap.
+Coverage is most complete over Europe (hey, we pay for this stuff!), where every descending and ascending acquisition 
+is acquired for both the A and B sensors. For most of India, however, only a single 12-day repeat cycle is possible, apart
+from the Himalayas, which are included in the seismic zoning, for which a best effort higher density revisit is programmed.
+
+### Sentinel-1 orbits, modes, swaths and scenes
+
+Sentinel-1 is a **polar orbiting** platform, i.e. after crossing the North Pole, it descends to the South Pole, crosses it and then ascends back to the North Pole. This explains the *orbitProperties_pass* metadata ASCENDING (ASC) and DESCENDING (DESC) property that is tagged to each scene in the S1_GRD_FLOAT catalog. The total time needed to go from North to South Pole and back is about 98 minutes (175 orbits in 12 days). In that time, the Earth is turning about 23 degrees to the East, which is why DESC orbits are slightly rotated towards the south-west, and ASC orbits towards the north-east. The *orbitNumber_start* metadata tag list the absolute orbit sequence number (counted from start of acquisitions). The *relativeOrbitNumber_start* is often more useful, as it provides the sequence number of the orbit in the 12 day revisit cycle. Scenes from the same *relativeOrbitNumber_start* are exactly 12 days apart (6 days if you combine A and B) and have (almost) the same SAR view configuration. 
```

Comment:
is tagged to each scene in the S1_GRD_FLOAT catalog -> is set on each scene in the S1_GRD_FLOAT colelction.

Reply:
revised